OpenSim/Common/DataTable.h

```diff
@@ -843,8 +843,9 @@ class DataTable_ : public AbstractDataTable {
         const size_t totalWidth{std::accumulate(columnWidths.cbegin(),
                                                 columnWidths.cend(),
                                                 static_cast<size_t>(0))};
-        result.reserve((totalWidth * table.size() * excessAllocation) +
-                       result.capacity());
+        result.reserve(result.capacity() +
+            static_cast<unsigned>(std::roundf(
+                totalWidth * table.size() * excessAllocation)));
```

Comment:
This is not based on public reference. It is just my approximate calculation to make sure to over-allocate. Didn't want to go work out the exact amount of space needed.

I don't have anything to add in this issues. Feel free to do edit any way you think is helpful/necessary. :+1: